web/src/app/lists/QueryList.tsx

```diff
@@ -79,6 +84,8 @@ export interface QueryListProps extends ControlledPaginatedListProps {
   mapVariables?: (vars: OperationVariables) => OperationVariables
 }
 
+export type QueryListProps = Omit<_QueryListProps, 'items'>
```

Comment:
Nice 😎, this tripped me up a couple of times working in Typescript with some of the QueryLists in the past